styles.css

```diff
@@ -1,60 +1,162 @@
-*{
-  margin: 0;
-  padding: 0;
-
-}
-
-.top-nav {
-  background-color: #1C1A19;
-  display: flex;
-  justify-content: space-between;
-    
-
-}  
-.left {
-  color: #FFFFFF;
-  padding: 15px 150px;
-  font-family: 'Inter';
-  font-style: normal;
-
-} 
-.right {
-    padding: 15px 150px;
-    line-height: 20px;
-}
-.right a{
-  color: #FFFFFF;
-  padding: 10px;
-  text-decoration: none;
-  
-  
-    
-} 
-.right i{
-  color: #FFFFFF;
-  text-decoration: none;
-  
-}
-.main {
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  grid-auto-rows: 1fr;
-  border: 1px solid green;
-  padding: 250px 50px;
-  color: #FFFFFF;
-  background-color: black;
-  /* display: flex;
-  align-items: center;
-  justify-content: space-between;
-  
-  
- 
-  
-  
-}
-.left-side ul li {
-  list-style-type: none;
-  
-  
-  
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #191765;
+  width: 100vw;
+  height: 100vh;
+}
+
+.header {
+  width: 100%;
+  height: max-content;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 30px;
+}
+
+.nav-links {
+  display: none;
+}
+
+.logo {
+  text-decoration: none;
+  color: #fff;
+  text-transform: capitalize;
+  font-size: 20px;
+}
+
+.fa-bars {
+  color: #fff;
+  font-size: 20px;
+}
+
+.main {
+  width: 90%;
+  margin: 50px auto 950px auto;

```

Comment:
To match the design provide let's use replace these two lines with `padding: 60px 24px` that should match the design.